### ISSUE TITLE
[Multicolumn] Image alt text defaults to column title

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#395]
## Issue summary

In the Multicolumn section, if an image is added, its `alt` text is automatically set from the column title. Depending on the intended purpose of the image/content, this could result in incorrect `alt` content or missing `alt` content.

<details>
  <summary>Screenshots</summary>

<img width="422" alt="Dawn theme multicolumn section. Chrome devtools displays alt text for image logo which matches column title text." src="https://user-images.githubusercontent.com/1392632/129215631-50cc596f-8191-495c-b707-be834aba87fa.png">

</details>

#### HTML

```html
<img … alt="Google Test" … />
<h3 …>Google Test</h3>
```

## Steps to reproduce

1. Go to a Dawn homepage which features the Multicolumn section with images with any supported browser
2. In admin, set the image `alt` text different than the column title text
3. Use the web inspector to review the image and heading markup

### Expected

- Image `alt` value to be as set in Admin.

### Actual

- Image `alt` is set as the column title text.

## Recommendation

Separate the image `alt` text from the column heading text. Use the liquid variable from the `image` object and set as the image `alt` attribute value.

 ### Recommended code

#### HTML

```html
<img … alt="Google" … />
<h3 …>Google Test</h3>
``` 

<!-- #### Diff

```diff
- line removed
+ line added
``` -->

## Specifications

- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- **Severity:** High
- **Effort:** Low

**Comment by robertKegel** [comment]
Changed the multicolumn.liquid section file to assign  block.settings.image.alt  to the img tag's alt attribute.
